main.tf

```
@@ -169,6 +169,7 @@ module "dns" {
   prefix     = var.prefix
 
   address  = module.loadbalancer.address
+  project  = local.dns_project
```

per the other comment for the locals block

```suggestion
  project  = local.rendered_dns_project
```